### DISCUSSION TITLE
Profile Under Review

### DISCUSSION THREAD

**Discussion by aditi2906** [github/india#659] [Students]
My profile continues to be under review even though I had registered for the event timely. I even mailed the authorities regarding the same but received no reply. Kindly assist in the matter.

**Comment by aditi2906** [comment]
@punitjain03 Sir, 
kindly assist in the issue raised above. It has now been more than a week and the profile continues to be in the under-review phase.

**Comment by Prathyusha-Guduru** [comment]
Same issue here! Please look into it.

**Comment by SaiManikantaKarnati** [comment]
Same issue here. Please look into it

**Comment by skmodi649** [comment]
@punitjain03 sir, I had applied for the program on the very first day, still I its showing under review, I request you to look into it, please

**Comment by Preetham-Reddy-007** [comment]
Same issue here

**Comment by Anup-Narkhede** [comment]
Same issue here, I have filled complete form, still its showing 'profile under review looks like you have not filled the complete form', I request you to look into it.
